README.md

```diff
@@ -175,8 +175,12 @@ Options:
     -h
     --help              Print this help.
 
+
+    --install-dir=PATH  Install into the specified PATH directory instead of the system-wide
```

Comment:
I think you should update the usage text in the `opm` script as well. They should be in sync.